pkg/testing/v1alpha1/service.go

```diff
@@ -63,10 +62,20 @@ func ServiceWithoutNamespace(name string, so ...ServiceOption) *v1alpha1.Service
 	for _, opt := range so {
 		opt(s)
 	}
-	s.SetDefaults(context.Background())
 	return s
 }
 
+// DefaultService creates a service with ServiceOptions and with default values set
+func DefaultService(name, namespace string, so ...ServiceOption) *v1alpha1.Service {
+	s := Service(name, namespace, append(so,WithServiceDefaults)...)
```

Comment:
Now that you have combined it all into one line you don't need a variable or separate return statement. 

```suggestion
	return Service(name, namespace, append(so,WithServiceDefaults)...)
```

Reply:
That is actually logically wrong